### ISSUE TITLE
no css file after update `simplebar-react` to v2.4.2

### ISSUE THREAD

**Issue by achmadk** [Grsmto/simplebar#621]
**Describe the bug**
I got an error when building my web app like this.

```sh
[vite-plugin-pwa:build] [vite]: Rollup failed to resolve import "simplebar-react/dist/simplebar.min.css" from "src/main.ts".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
```

I check file `[my_web_app_directory]/node_modules/simplebar-react/dist/simplebar.min.css` is missing after update `simplebar-react` to v2.4.2

**To Reproduce**
1. Install `simplebar-react` v2.4.2.
2. Add `import 'simplebar-react/dist/simplebar.min.css'` into your web app entry point.
3. See an error from `webpack`/ `vite`/ etc.

**Expected behavior**
File `simplebar-react/dist/simplebar.min.css` is not missing and correctly imported into the web app.

**Reproducible example**

**Additional context**

**Your environment**

| Software         | Version(s) |
| ---------------- | ---------- |
| SimpleBar (react)        | 2.4.2
| Browser          | any browser
| Operating System | Ubuntu 22.04.1

**Comment by exilvm** [comment]
You are right. I'm guessing it's caused by the recent changes to the Travis build.

**Comment by GuichiZhao** [comment]
which is the latest working version?

**Comment by achmadk** [comment]
> which is the latest working version?

use `simplebar-react` v2.4.1 as temporary solution @GuichiZhao .

**Comment by Grsmto** [comment]
Hey, nothing has changed since the last version, it's the README that is pointing to the wrong CSS file. 
As a workaround you can do `import 'simplebar/dist/simplebar.min.css'` for now.
I will send a patch for this asap.

**Comment by Grsmto** [comment]
Solved in [simplebar-react@2.4.3](https://github.com/Grsmto/simplebar/releases/tag/simplebar-react%402.4.3)

**Comment by ZeeshanAhmadKhalil** [comment]
I am getting 

`Module not found: Can't resolve 'simplebar/dist/simplebar.min.css'`

Version
`"simplebar-react": "^3.0.0"`
